ci-operator/step-registry/load-balancer/load-balancer-commands.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
Note: I put this step outside of `openstack` directory because this is meant to be used by any platform.
This step is made to be working on any platforms who want to, and we will support platform specific bits like in https://github.com/openshift/release/pull/36042/files#diff-2a2603e0513683396c162558d9681c7f1bb429f0fddfe28b6abc5459c82b8d68R31